Applications/IntelliDivide/Contracts/Statistics/EdebandUsage.cs

```diff
@@ -42,7 +42,7 @@ public class EdgebandUsage : IExtensibleDataObject, IContainsUnitSystemDependent
         /// Total length used for edgebands of this type (meters or feet)
         /// </summary>
         [JsonProperty(Order = 6)]
-        [ValueDependsOnUnitSystem(BaseUnit.Meter)]
+        [ValueDependsOnUnitSystem(BaseUnit.Meter, DecimalPrecision.TwoDecimalPlaces, DecimalPrecision.TwoDecimalPlaces)]
```

Comment:
I guess for "Meter" it is always 2 decimal places.

Reply:
Yes, but I set 1 (metric) and 3 (imperial) as defaults as they are used way more often.